Migration.md

```diff
@@ -12,6 +12,20 @@ release will remove the deprecated code.
 
 * Header files under `ignition/...` are deprecated and will be removed in future versions.
   Use `gz/...` instead.
+  
+* The following `IGN_` prefixed macros are deprecated and will be removed in future versions.
```

Comment:
it looks like these macros are only available if you use the `ignition/*`  include files and are already unavailable with `gz/*` include files. If this is the intended approach, then we should reword this part of the migration guide.

otherwise, we should add the macros as deprecated in the `gz/*` include files

Reply:
Ah, alright, I'll reword it, for the other repos as well. The same approach was used for everything.

Reply:
https://github.com/gazebosim/gz-utils/pull/64/commits/03652cb86a35957c1ee9ec030ffce49bde4841c8